config/orte_check_lsf.m4

```diff
@@ -120,6 +120,71 @@ AC_DEFUN([ORTE_CHECK_LSF],[
                         [orte_check_lsf_happy="yes"],
                         [orte_check_lsf_happy="no"])])
 
+
+          # Some versions of LSF ship with a libevent.so in their library path.
+          # This is _not_ a copy of Libevent, but something specific to their project.
+          # The Open MPI components should not need to link against LSF's libevent.so
+          # However, the presence of it in the linker search path can cause a problem
+          # if there is a system installed Libevent and Open MPI chooses the 'external'
+          # event component prior to this stage.
+          #
+          # Add a check here to see if we are in a scenario where the two are conflicting.
+          # In which case the earlier checks for successful compile of an LSF program will
+          # have failed with messages like:
+          #   lib64/libevent_pthreads.so: undefined reference to `evthread_set_condition_callbacks'
+          #   lib64/libevent_pthreads.so: undefined reference to `event_mm_malloc_'
+          #   lib64/libevent_pthreads.so: undefined reference to `event_mm_free_'
+          #   lib64/libevent_pthreads.so: undefined reference to `evthread_set_id_callback'
+          #   lib64/libevent_pthreads.so: undefined reference to `evthread_set_lock_callbacks'
+          # Because it picked up -levent from LSF, but -levent_pthreads from Libevent.
+          #
+          # So look for a function that libevent_pthreads is looking for from libevent.so
+          # If it does not appear here when adding the same option then we have
+          # a conflict.
+          orte_check_lsf_event_conflict=na
+          AS_IF([test "$orte_check_lsf_happy" = "no"],
+                [AS_IF([test "$opal_event_external_support" = "yes"],
+                       [AS_IF([test "$orte_check_lsf_libdir" = "" ],
+                              [],
+                              [LDFLAGS="$LDFLAGS -L"$orte_check_lsf_libdir])
```

Comment:
That's a weird placement for the closing quote -- is there a reason to have it there?

Reply:
nope. I can fix.